lldb/test/API/functionalities/data-formatter/swift/string-index/TestSwiftStringIndexFormatters.py

```diff
@@ -0,0 +1,144 @@
+# TestSwiftStringIndexFormatters.py
+#
+# This source file is part of the Swift.org open source project
+#
+# Copyright (c) 2014 - 2016 Apple Inc. and the Swift project authors
+# Licensed under Apache License v2.0 with Runtime Library Exception
+#
+# See https://swift.org/LICENSE.txt for license information
+# See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+#
+# ------------------------------------------------------------------------------
```

Comment:
@augusto2112 what about this notice?

Reply:
You can remove it too, sorry I didn't highlight it :)